src/components/Dashboard/Dashboard.js

```diff
@@ -22,6 +72,37 @@ export const postList = [
 ];
 
 export const Dashboard = () => {
+  const [kpiListState, setKpiListState] = useState([]);
+
+  useEffect(() => {
+    const campaings = kpiList.Campaings;
+    const subscribers = kpiList.Subscribers;
+
+    setKpiListState({
+      campaings,
+      subscribers,
+    });
+  }, []);
+
+  const renderKpis = (listType) => {
+    return kpiListState[listType]?.map((kpi, index) => (
+      <Kpi
+        key={`kpi-campaing-${index}`}
+        iconClass={kpi.iconClass}
+        kpiValue={kpi.kpiValue}
+        kpiTitle={kpi.kpiTitle}
+        kpiPeriod={kpi.kpiPeriod}
+        active={kpi.active}
+        colSize={kpiListSize.campaigns}
```

Comment:
This way the function always take the size for campaigns, what happens if we have more kpis for subscribers?

Reply:
You could add this size as parameter into `renderKpis` function.